src/test/java/it/gov/pagopa/idpay/transactions/repository/RewardTransactionRepositoryTest.java

```diff
@@ -0,0 +1,55 @@
+package it.gov.pagopa.idpay.transactions.repository;
+
+import it.gov.pagopa.idpay.transactions.BaseIntegrationTest;
+import it.gov.pagopa.idpay.transactions.model.RewardTransaction;
+import it.gov.pagopa.idpay.transactions.test.fakers.RewardTransactionFaker;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.DirtiesContext;
+import reactor.core.publisher.Flux;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@DirtiesContext
+class RewardTransactionRepositoryTest extends BaseIntegrationTest {
+
+    @Autowired
+    protected RewardTransactionRepository rewardTransactionRepository;
+
+    @AfterEach
+    void clearData(){
+        rewardTransactionRepository.deleteAll().block();
```

Comment:
don't clear all data if you think to write a TestIntegrated! Otherwise you will erase all the data in that environment!
just delete all data that you have created, using some known filter configured taking care also of the available indexes